database/migrations/2022_11_12_103015_index_user_id_on_beatmapset_nominations.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the GNU Affero General Public License v3.0.
+// See the LICENCE file in the repository root for full licence text.
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class IndexUserIdOnBeatmapsetNominations extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('beatmapset_nominations', function (Blueprint $table): void {
+            $table->index('user_id');
```

Comment:
adding reset as well is probably good idea. Also the rollback should have the columns in an array